pkg/remote/aws/ec2_internet_gateway_enumerator.go

```diff
@@ -0,0 +1,49 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+	tf "github.com/cloudskiff/driftctl/pkg/remote/terraform"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+type EC2InternetGatewayEnumerator struct {
+	repository     repository.EC2Repository
+	factory        resource.ResourceFactory
+	providerConfig tf.TerraformProviderConfig
```

Comment:
We do not need `providerConfig` here, it was used in s3 enumerators to get the current region but we don't care here. You can remove it

Reply:
Nice catch! I will fix the other EC2 Enumerators then.